### PR TITLE
adds and implements dropdown search

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

#766 

## Purpose/Implementation Notes

Adds a dropdown search components and implements it on the compendia download page.

## Types of changes

* [ ] Bugfix (non-breaking change which fixes an issue)
* [X] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests

Tested on chrome and ff

## Checklist

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)
* [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

Screen Shot 2020-01-03 at 11 22 32 AM

### CODE REVIEW

**Comment by davidsmejia** [comment]
Actually this is not quite ready to merge but a review would be good at this point

File: src/pages/Compendia/DownloadCompendia.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ReactGA from 'react-ga';
 import { formatSentenceCase, formatBytes } from '../../common/helpers';
 import InfoBadge from '../../common/icons/info-badge.svg';
 
-import Dropdown from '../../components/Dropdown';
+import Dropdown from '../../components/DropdownSearch';
```
Comment (arielsvn) [inline src/pages/Compendia/DownloadCompendia.js]: call this `DropdownSearch`?

File: ' [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```
Comment (arielsvn) [inline ']: what's this file?

File: src/components/DropdownSearch.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+              className={classnames('dropdown-search__dropdown-item', {
+                'dropdown-search__dropdown-item__highlight': isHighlighted(
+                  option
+                ),
+              })}
+              key={label(option)}
+              ref={el => {
+                if (!isHighlighted(option)) return null;
+                highlightedRef.current = el;
+                return highlightedRef;
```
Comment (arielsvn) [inline src/components/DropdownSearch.js]: do we need to return here?

**Comment by arielsvn** [comment]
@dvenprasad this could also use your input [demo here](https://deploy-preview-819--refinebio.netlify.com/compendia#normalized). Personally, I'm not a fan of this type of control to select the compendia.

**Comment by dvenprasad** [comment]
@arielsvn can you elaborate? 

@davidsmejia I probably should have been clearer yesterday by what I meant (also helps that i had some time to think about it) 

Screen Shot 2020-01-03 at 11 52 37 AM

(ignore the random `text` in the middle there)

File: src/pages/Compendia/DownloadCompendia.js (hunk 1 above)
Comment (davidsmejia) [reply src/pages/Compendia/DownloadCompendia.js]: 👍

**Pull request merged**